bokeh/properties.py

```diff
@@ -104,7 +104,14 @@ def value(val):
     '''
     return dict(value=val)
 
-bokeh_integer_types = (np.int8, np.int16, np.int32, np.int64) + integer_types
+bokeh_bool_types = (bool,)
+try:
+    import numpy as np
+    bokeh_bool_types += (np.bool8,)
+except ImportError:
+    np = None
```

Comment:
I think this can just `pass`? Also maybe better to do inline to be consistent with `DataTime`? 

https://github.com/jakirkham/bokeh/blob/fix_bokeh_2893/bokeh/properties.py#L1058